src/schema/v2/partner/partnerInquirerCollectorProfile.ts

```diff
@@ -28,24 +28,3 @@ export const PartnerInquirerCollectorProfile: GraphQLFieldConfig<
   type: InquirerCollectorProfileType,
   description: "Inquiry requester's profile",
 }
-
-export const InquiryRequestType = new GraphQLObjectType<any, ResolverContext>({
-  name: "PartnerInquiryRequest",
-  fields: {
-    collectorProfile: {
-      type: InquirerCollectorProfileType,
-      resolve: (
-        { id: inquiryId, partnerId },
-        _args,
-        { partnerInquirerCollectorProfileLoader }
-      ) => {
-        if (!partnerInquirerCollectorProfileLoader) return
-
-        return partnerInquirerCollectorProfileLoader({
-          partnerId,
-          inquiryId,
-        })
-      },
-    },
-  },
-})
```

Comment:
This was moved to its separate file since the name scope is not only relevant to the Collector Profile.

Reply:
I don't know why the Collector Profile needs to be part of this since we can access that separately, but that is out of scope at the moment.